catalog/core/catalog-core-camelcomponent/src/test/java/ddf/camel/component/catalog/CatalogComponentTest.java

```diff
@@ -343,16 +271,7 @@ private InputTransformer getTransformer(String mimeType, String id) throws Excep
 
     ServiceReference[] refs = null;
     try {
-      String filter =
-          "(&("
-              + MimeTypeToTransformerMapper.MIME_TYPE_KEY
-              + "="
-              + mimeType
-              + ")("
-              + MimeTypeToTransformerMapper.ID_KEY
-              + "="
-              + id
-              + "))";
+      String filter = "(&(" + "mime-type" + "=" + mimeType + ")(" + "id" + "=" + id + "))";
```

Comment:
Why not `String.format("(&(mime-type=%s)(id=%s))", mimeType, id);`?

Reply:
done